FancyLabel - iOS/MFLReactFancyLabel.m

```diff
@@ -39,11 +39,13 @@ - (void)dealloc
 
 - (void)addSubview:(UIView *)view
 {
-    if ([view respondsToSelector:@selector(textStorage)]) {
-        self.observedView = view;
-        view.hidden = YES;
+    if ([view isKindOfClass:[THLabel class]]) {
```

Comment:
Should this be changed to just `UILabel` to allow for wider changes in the future and not limit ourselves to a subclass?

Reply:
Also, I would like to change it entirely to just exclude `RCTTextView` objects, but I kept running into define issues